### PR TITLE
[DDW-863]: Implement main page design story

### PR DESCRIPTION
**Pull request by yakovkaravelov**
This PR implemented main page design story.
[YouTrack Ticket](https://iohk.myjetbrains.com/youtrack/issue/DDW-863)

## Todos
- [x] Implement main page design story

## Screenshots
image
image
image

## Testing Checklist
- [Slack QA thread](https://input-output-rnd.slack.com/archives/GGKFXSKC6/p1570459933118600)
- [x] Go to `Pages` => `Main Page` story on storybook and check if it matches with wireframe

### Basics

- [ ] PR has been assigned and has appropriate labels (`feature`/`bug`/`chore`, `release-x.x.x`)
- [ ] PR is updated to the most recent version of the target branch (and there are no conflicts)
- [ ] PR has a good description that summarizes all changes and shows some screenshots or animated GIFs of important UI changes
- [ ] CHANGELOG entry has been added to the top of the appropriate section (*Features*, *Fixes*, *Chores*) and is linked to the correct PR on GitHub
- [ ] Automated tests: All acceptance and unit tests are passing (`yarn test`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *development* build (`yarn dev`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *production* build (`yarn package` / CI builds)
- [ ] There are no *flow* errors or warnings (`yarn flow:test`)
- [ ] There are no *lint* errors or warnings (`yarn lint`)
- [ ] There are no *prettier* errors or warnings (`yarn prettier:check`)
- [ ] There are no missing translations (running `yarn manage:translations` produces no changes)
- [ ] Text changes are proofread and approved (Jane Wild / Amy Reeve)
- [ ] Japanese text changes are proofread and approved (Junko Oda)
- [ ] UI changes look good in all themes (Alexander Rukin)
- [ ] Storybook works and no stories are broken (`yarn storybook`)
- [ ] In case of dependency changes `yarn.lock` file is updated

### Code Quality
- [ ] Important parts of the code are properly commented and documented
- [ ] Code is properly typed with flow
- [ ] React components are split-up enough to avoid unnecessary re-renderings
- [ ] Any code that only works in main process is neatly separated from components

### Testing
- [ ] New feature/change is covered by acceptance tests
- [ ] New feature/change is manually tested and approved by QA team
- [ ] All existing acceptance tests are still up-to-date
- [ ] New feature/change is covered by Daedalus Testing scenario
- [ ] All existing Daedalus Testing scenarios are still up-to-date

### After Review
- [ ] Merge the PR
- [ ] Delete the source branch
- [ ] Move the ticket to `done` column on the YouTrack board
- [ ] Update Slack QA thread by marking it with a green checkmark

### CODE REVIEW

File: source/features/blocks/components/BlockList.scss [hunk 1]
```diff
@@ -0,0 +1,87 @@
+.blockListContainer {
+  width: 872px;
```
Comment (DeeJayElly) [inline source/features/blocks/components/BlockList.scss]: Alphabetic ordering

File: source/features/epochs/components/EpochList.scss [hunk 2]
```diff
@@ -1,7 +1,12 @@
 .epochListContainer {
   width: 872px;
```
Comment (DeeJayElly) [inline source/features/epochs/components/EpochList.scss]: Alphabetic ordering

File: source/features/widgets/footer/components/Footer.scss [hunk 3]
```diff
@@ -8,13 +8,18 @@
 }
 
 .footerContainer {
-  background-color: $primary-bg;
-  padding: 78px 40px;
+  width: 100%;
+
+  .footerContent {
+    width: 872px;
```
Comment (DeeJayElly) [inline source/features/widgets/footer/components/Footer.scss]: Alphabetic ordering

File: source/features/blocks/components/BlockList.scss (hunk 1 above)
Comment (yakovkaravelov) [reply source/features/blocks/components/BlockList.scss]: This is WIP pr. Need to update several components, will do this sorting once they are done.

File: source/features/epochs/components/EpochList.scss (hunk 2 above)
Comment (yakovkaravelov) [reply source/features/epochs/components/EpochList.scss]: This is WIP pr. Need to update several components, will do this sorting once they are done.

File: source/features/widgets/footer/components/Footer.scss (hunk 3 above)
Comment (yakovkaravelov) [reply source/features/widgets/footer/components/Footer.scss]: This is WIP pr. Need to update several components, will do this sorting once they are done.

**Comment by daniloprates** [comment]
@yakovkaravelov please, let us know when's done for review.

**Comment by yakovkaravelov** [comment]
> @yakovkaravelov please, let us know when's done for review.

Sure.

**Comment by yakovkaravelov** [comment]
@daniloprates @DeeJayElly @a-rukin Please review this PR. It's ready for review.

**Comment by yakovkaravelov** [comment]
@daniloprates @DeeJayElly @a-rukin @gnpf Please review this.
This is done.

**Comment by yakovkaravelov** [comment]
@daniloprates @DeeJayElly @a-rukin @gnpf Please review this.

**Comment by IuliaDolishniak** [comment]
@yakovkaravelov when minimizing width table looks inconsistent
screen
when make the width narrower only 2 columns are visible
screen2

**Comment by yakovkaravelov** [comment]
> @yakovkaravelov when minimizing width table looks inconsistent
> screen
> when make the width narrower only 2 columns are visible
> screen2

For a bit smaller width screens, I will fix styles.
And regarding only 3 columns in very small screen, that's based on mobile design. https://app.zeplin.io/project/5878c3481bb7631c20cb161a/screen/5d63b15b38345e9b5a6d7de0
Will update soon.

**Comment by IuliaDolishniak** [comment]
When resizing the page vertical and horizontal scroll bars appear
Image 3

**Comment by yakovkaravelov** [comment]
> When resizing the page vertical and horizontal scroll bars appear
> Image 3

That's because it's storybook, if you test this in actual app, no scrollbar when resizing window. @IuliaDolishniak

**Comment by IuliaDolishniak** [comment]
Great job @yakovkaravelov

**Pull request merged**